rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -536,7 +536,8 @@ class LockingCapTest {
 
         private LockingCapSupport lockingCapSupport;
         private BridgeSupport bridgeSupport;
-        private final LockingCapConstants constants = LockingCapMainNetConstants.getInstance();
+        private final BridgeConstants bridgeMainNetConstants = BridgeMainNetConstants.getInstance();
+        private final LockingCapConstants constants = bridgeMainNetConstants.getLockingCapConstants();
```

Comment:
I wouldn't do this. It's just trying to force something. The tests we discussed adding should cover for this